viewer/packages/tools/src/Measurement/types.ts

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+/**
+ * Custom callback for users to change measurement label content.
+ */
+export type DistanceToLabelDelegate = (distanceInMeters: number) => string;
+
+/**
+ * Delegate for measurement added events.
+ */
+export type MeasurementAddedDelegate = () => void;
```

Comment:
This delegate must have an event argument that contains the `Measurement` added.
`(event: { measurement: Measurement }) => void`.

Reply:
Also, the Delegate-types are not properly exported to public API I believe. This is easy to see by looking at API reference docs in docs preview.